test/integration/nodemanagement/cluster_options.bats

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bats
+
+load ../helpers
+
+@test "cluster options" {
+	# cluster option swarm.overcommit
+	swarm_manage --cluster-opt swarm.overcommit=-2 nodes://192.168.56.22:4444
```

Comment:
To run integration tests on your machine:
./test/integration/run.sh

You may also try `bats test/integration/nodemanagement/cluster_options.bats` to rule out obvious problem first.